src/pages/guides/api/eventsingress_api.md

```diff
@@ -13,6 +13,12 @@ title: Events Publishing API
    * create your own `Custom Events Provider`
    * create at least one `Event Metadata` associated with the above
 
+<InlineAlert slots="title, text"/>
+
+Data Residency:
+
+Please note that all custom events are stored in the **United States (US)** region.  
+
```

Comment:
Title is still not rendered. Shall we remove the placeholder then?
```suggestion
<InlineAlert slots="text"/>

Please note that all custom events are stored in the **United States (US)** region.  
```

Reply:
I was hoping to keep it. The header will start appearing once gatsby issue is fixed.

Reply:
But in that case the slot name is called `text1` and not `text`, so, I am not sure if it will automatically start working.

Reply:
Thanks for clarifying.